doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -1,195 +1,340 @@
-.. _train-experiment-tracking:
+.. _train-experiment-tracking-native:
 
+===================
 Experiment Tracking
 ===================
 
-Most experiment tracking libraries work out-of-the box with Ray Train. This means
-you can use the regular tracking library APIs to report your results, models, and
-other artifacts.
+.. note::
+    This guide is relevant for all DataParallelTrainer, including TorchTrainer and TensorflowTrainer.
 
-Ray Train also exposes logging callbacks that automate some of these tasks.
+Most experiment tracking libraries work out-of-the-box with Ray Train. 
+This guide provides instructions on how to set up the code so that your favorite experiment tracking libraries 
+can work for distributed data parallel training with Ray Train. We will conclude the session with debugging
+tips.
 
-.. _train-experiment-tracking-native:
+Before we begin, the following is roughly how you can use the native experiment tracking lirary calls 
+inside of Ray Train. 
 
-Using native experiment tracking libraries
-------------------------------------------
+.. code-block:: python
 
-.. tab-set::
+    from ray.train.torch import TorchTrainer
+    from ray.train import ScalingConfig
 
-    .. tab-item:: PyTorch
+    def train_func(config):
+        # Training code and native lexperiment tracking library calls go here.
 
-        You can use experiment tracking libraries such as Weights & Biases, Mlflow, or
-        Comet directly in your Ray Train training loop.
+    scaling_config = ScalingConfig(num_workers=2, use_gpu=True)
+    trainer = TorchTrainer(train_func, scaling_config=scaling_config)
+    result = trainer.fit()
 
-        There are two things to keep in mind:
+Ray Train lets you use native experiment tracking libraries by customizing the tracking 
+logic inside the ``train_func`` function. In this way, you can port your experiment tracking 
+logic to Ray Train with minimal changes. 
 
-        1. Your code is executed in parallel on many workers. However, you often only want to report
-        results from one of these workers (usually the first worker - the "rank 0" worker).
+Log distributed training experiments
+====================================
 
-        2. When using the native libraries, you should report the results to both Ray Train and
-        the experiment tracking library.
+In this section, we break the interaction with experiment tracking backend into 4 logical steps:
+- Set up to connect to a tracking backend
+- Configure and launch a run
+- Log
+- Finish the run
 
-        Example:
+Let's dive into each one of them.
 
-        .. code-block:: python
+.. note::
+
+    For some scenarios, every Worker generates an identical copy and saving a single copy is sufficient. 
+    Ray Train lets you apply logic to only the rank 0 worker with the following method:
+    :meth:`context.get_world_rank() <ray.train.context.TrainContext.get_world_rank>`.
```

Comment:
This needs to be more explicit, I'm not sure what "copy" is referring to here. 

Maybe having the code snippet first will help make this clear what this is referring to.

Reply:
this is updated.